Dockerfile

```diff
@@ -9,5 +9,5 @@ ADD scm-source.json /scm-source.json
 EXPOSE 8080
 
 # run the server when a container based on this image is being run
-ENTRYPOINT java -jar -Djava.security.egd=file:/dev/./urandom nakadi.jar
+ENTRYPOINT java $JAVA_OPTS -jar -Djava.security.egd=file:/dev/./urandom nakadi.jar
```

Comment:
@v-stepanov from my limited knowledge of JVM it's just a matter of exporting an environment variable, like `JAVA_OPTS='-Djava.awt.headless=true'` and new java processes would use it. Makes sense?

Then one could pass these variables as parameters to the container.